docs/get-started/index.md

```diff
@@ -6,3 +6,24 @@ title: Get started
 Ministry of Justice builds on the principles and guidelines published by GDS. The guidance here is intended to supplement that published by GDS and not replace it.
 
 The guidelines published here show how these have been adapted and refined to fit the goals and context of MOJ Digital & Technology.
+
+  <div class="app-main-wrapper govuk-main-wrapper govuk-main-wrapper--l">
+
+    <div class="govuk-grid-row">
+
+      <div class="govuk-grid-column-one-third govuk-!-margin-bottom-8">
+        <h2 class="govuk-heading-l">Components</h2>
+        <p>Save time with reusable, accessible components for forms, navigation, panels, tables and more.</p>
+        <p><a class="app-link govuk-!-font-weight-bold" href="/components">Browse components</a></p>
+      </div>
+
+      <div class="govuk-grid-column-one-third govuk-!-margin-bottom-8">
+        <h2 class="govuk-heading-l">Patterns</h2>
+        <p>Help users complete common tasks like entering names and addresses, filling in forms and creating accounts.</p>
+        <p><a class="app-link govuk-!-font-weight-bold" href="/patterns">Browse patterns</a></p>
```

Comment:
As above, the links need to be relative:
```suggestion
        <p><a class="app-link govuk-!-font-weight-bold" href="./components">Browse components</a></p>
      </div>

      <div class="govuk-grid-column-one-third govuk-!-margin-bottom-8">
        <h2 class="govuk-heading-l">Patterns</h2>
        <p>Help users complete common tasks like entering names and addresses, filling in forms and creating accounts.</p>
        <p><a class="app-link govuk-!-font-weight-bold" href="./patterns">Browse patterns</a></p>
```

Reply:
Done